run.go

```diff
@@ -8,31 +8,41 @@ import (
 // Start will launch the go routines in the background to perform Autopilot.
 // When the context passed in is cancelled or the Stop method is called
 // then these routines will exit.
-func (a *Autopilot) Start(ctx context.Context) {
+func (a *Autopilot) Start(ctx context.Context) error {
```

Comment:
What is the `error` return value supposed to be used for? We only `return nil` currently.

Reply:
Bad refactor. Originally all I did was make start return an error if it wasn't in the NotRunning state instead of allowing starting it again even though the state was ShuttingDown. After allowing it to be started while the previous was shutting down there was no reason for the error.